packages/transaction-messages/src/__typetests__/transaction-message-typetests.ts

```diff
@@ -51,8 +53,9 @@ setTransactionMessageLifetimeUsingBlockhash(
     mockBlockhashLifetime,
     null as unknown as Extract<TransactionMessage, { version: 0 }>,
     // @ts-expect-error Version should match
-) satisfies Extract<Transaction, { version: 'legacy' }> & ITransactionWithBlockhashLifetime;
+) satisfies Extract<TransactionMessage, { version: 'legacy' }> & ITransactionMessageWithBlockhashLifetime;
 
+<<<<<<< HEAD
```

Comment:
![Added via Giphy](https://media4.giphy.com/media/3U3CCRZLOdav41NYaJ/giphy.gif?cid=c9487743onvcg5jwdnd7tqk3qo4wrcpd52udike6zspet3bq&ep=v1_gifs_search&rid=giphy.gif&ct=g)

Reply:
Probably why `gt modify` broke and/or was broken by `gt modify` 😅